src/libraries/Microsoft.PowerFx.Core/Syntax/UserDefinitions.cs

```diff
@@ -103,21 +96,9 @@ private IEnumerable<UserDefinedFunction> CreateUserDefinedFunctions(IEnumerable<
                 userDefinedFunctions.Add(func);
             }
 
-            BindUserDefinedFunctions(userDefinedFunctions, ReadOnlySymbolTable.NewDefault(texlFunctionSet) as INameResolver, errors);
```

Comment:
User defined functions shouldn't be bound on creation, this will result in double binding, instead hosts can bind body by calling UserDefinedfunction.BindBody()